eventuate-tram-producer-jdbc/src/main/java/io/eventuate/tram/messaging/producer/jdbc/TramMessageProducerJdbcConfiguration.java

```diff
@@ -3,12 +3,15 @@
 import io.eventuate.javaclient.spring.jdbc.EventuateSchema;
 import io.eventuate.javaclient.spring.jdbc.IdGenerator;
 import io.eventuate.javaclient.spring.jdbc.IdGeneratorImpl;
+import io.eventuate.local.java.kafka.producer.EventuateKafkaProducerConfigurationProperties;
 import io.eventuate.tram.messaging.producer.MessageProducer;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
+@EnableConfigurationProperties(EventuateKafkaProducerConfigurationProperties.class)
```

Comment:
Does `TramMessageProducerJdbcConfiguration` need those properties? it writes to the database.

Reply:
Fixed. Tested.